src/core/core.scale.js

```diff
@@ -537,6 +541,38 @@ module.exports = Element.extend({
 		return rawValue;
 	},
 
+	_parseValue: function(raw) {
+		var value = +this.getRightValue(raw);
+		var start, end, isarr;
+
+		if (helpers.isArray(value)) {
+			start = value[0];
+			end = value[1];
+			isarr = true;
+		} else {
+			start = 0;
+			end = value;
+			isarr = false;
+		}
+
+		return {
+			min: Math.min(start, end),
+			max: Math.max(start, end),
+			start: start,
+			end: end,
+			isarr: isarr
+		};
+	},
+
+	getScaleLabel: function(rawValue) {
+		var v = this._parseValue(rawValue);
+		if (v.issar === true) {
```

Comment:
lets keep as it is until unit tests will be done. I use this check currently only on puting labels. But i expect it maybe useful in future; We need this to seperate logic for tests between array and general presentation of vals. A lot of unit tests expects labels to be simple val not  this v.min : v.min + ' ; ' + v.max. So i had to introduce this to make those tests be not failed. Cause it would be not correct to change tests from my side i think.

Reply:
This typo i will change in next commit, thx for advising

Reply:
Changing it shouldn't cause any of the unit tests to fail though because none of the unit tests would have floating bar charts that define separate `min` and `max` values

Reply:
Oh, I see now. This is the same as the other case. We're setting `min` to `0` if it's not a floating bar. Okay, I suppose it's only the typo that should be changed then